stratum/hal/lib/barefoot/bf.proto

```diff
@@ -43,3 +43,12 @@ message BfrtDeviceConfig {
 
   repeated P4Program programs = 1;
 }
+
+// Internal message for action data passed from and to the SdeWrapper.
+message BfActionData {
+  message Param {
+    int32 id = 1;
+    bytes value = 2;
+  }
+  repeated Param params = 1;
+}
```

Comment:
Decide if/how to bundle internal data structure for communication from/to the SdeWrapper and managers.

Reply:
Comments from the sync:

- Having new data types is fine, if it significantly simplifies API usage

- There is some precedence for this in the BCM code with `BcmFlowEntry`